### PR TITLE
Update tracing task doc

### PR DESCRIPTION
**Pull request by bianpengyuan**
Change namespace from istio-control back to istio-system. Also update tracing screenshot since policy spans won't be included by default.

### CODE REVIEW

File: content/docs/tasks/telemetry/distributed-tracing/overview/index.md [hunk 1]
```diff
@@ -81,8 +81,8 @@ When you make downstream calls in your applications, make sure to include these
 
 ## Trace sampling
 
-Istio captures a trace for all requests by default. For example, when
-using the Bookinfo sample application above, every time you access
+When installing with demo profile, Istio captures a trace for all requests by default.
```
Comment (sdake) [inline content/docs/tasks/telemetry/distributed-tracing/overview/index.md]: `Istio captures a trace for all requests by default when installing with the demo profile.`

Comment (bianpengyuan) [reply content/docs/tasks/telemetry/distributed-tracing/overview/index.md]: Done.

**Comment by bianpengyuan** [comment]
@douglas-reid @sdake I updated the overview a bit. Please take another look. Thanks!

**Comment by geeknoid** [comment]
You have a few CI failures to take care of.

**Comment by bianpengyuan** [comment]
@geeknoid It's weird that lint thinks [Lightstep](../lightstep/) is not correctly spelled (with the text part highlighted). Changing the relative path to full path makes lint happy.

**Comment by geeknoid** [comment]
We don't support relative paths in the docs, so that was definitely part of the problem.

**Pull request merged**